src/Views/Admin/Admin.js

```diff
@@ -25,15 +26,21 @@ export default function Admin() {
   }
 
   async function handleSave(e) {
-    e.preventDefault();
-    const dog = { name: name, age: age, bio: bio, image: image, breed: breed };
-    await insertDog(dog);
+    try {
+      e.preventDefault();
+      const dog = { name: name, age: age, bio: bio, image: image, breed: breed };
```

Comment:
Not totally necessary to do this, we could just pass all these properties into the insert dog function without first creating a new object.